src/components/views/game/gamesummary/GameSummary.tsx

```diff
@@ -1,51 +1,41 @@
 import React from "react"
 import { observer } from "mobx-react-lite"
-import { useGames } from "../../../../hooks/api/useGames"
-import { extractId } from "../../../../util/extractId"
-import BaseContainer from "../../../ui/BaseContainer"
-import SendIcon from "@mui/icons-material/Send"
-import LogoutIcon from "@mui/icons-material/Logout"
-import Button from "@mui/material/Button"
-import { useNavigate } from "react-router-dom"
-import { Paths } from "../../../routing/routers/Paths"
 import ReactConfetti from "react-confetti"
```

Comment:
It currently is too large for the 1366x768 resolution:
![grafik](https://user-images.githubusercontent.com/1506818/170821037-6a4a4113-6fc3-42e0-ad6e-b34a10b8afb9.png)

Reply:
Fixed with https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/pull/129/commits/c82230369afddc98d00cf7fe04c3b76dc826e856

Reply:
yes it was broken in the merge/rebase from the other branch. I am currently working on fixing it

Reply:
I did a small rework on the branch.
this is how it looks now
![Bildschirmfoto am 2022-05-28 um 14 55 22](https://user-images.githubusercontent.com/91435500/170826516-3b469765-63ee-4116-adb1-b5f79437474a.png)